<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="taskViewer_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ObelixPro</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Диспетчер задач</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keyScaner_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ObelixPro</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сканер 
клавиатуры</string>
    </property>
   </widget>
   <widget class="QPushButton" name="peFiles_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ObelixPro</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Просмотр 
PE/ELF-файлов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ObelixPro</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите один из пунктов:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
